_layouts/bib.html

```diff
@@ -25,6 +25,16 @@
               {% assign author_is_self = true %}
             {% endif %}
           {% endif %}
+          {% assign coauthor_url = "" %}
+          {% if site.data.coauthors[author.last] %}
+            {% for db_author in site.data.coauthors[author.last] %}
+              {% if db_author.firstname contains author.first %}
+                {% assign coauthor_url = db_author.url %}
```

Comment:
Minor stylistic change:
```suggestion
            {% for coauthor in site.data.coauthors[author.last] %}
              {% if coauthor.firstname contains author.first %}
                {% assign coauthor_url = coauthor.url %}
```